File: kuku/kuku.java
public class kuku {
    public static void main(String[] args) {

        for (int i = 1; i < 10; i++) {
            for (int j = 1; j < 10; j++) {
                String str = String.format("%2d", i * j);

                System.out.print(str);
                System.out.print(" ");
            }
            System.out.println("");
        }
    }
}

File: kuku/kuku.py

def check(j):
    if (j % 9 == 0):
        print("\n")


def roop():
    for i in range(1, 10):
        for j in range(1, 10):
            print('{:2}'.format(i*j), end=" ")

          #  print(i * j, end=" ")
            check(j)


roop()
